src/main/java/jacchm/footballapp/repository/StandingInputRepository.java

```diff
@@ -0,0 +1,13 @@
+package jacchm.footballapp.repository;
+
+import jacchm.footballapp.model.entity.StandingsInput;
+import org.springframework.data.repository.CrudRepository;
+
+import java.util.List;
+import java.util.Optional;
+
+public interface StandingInputRepository extends CrudRepository<StandingsInput, Integer> {
+
+    Optional<List<StandingsInput>> findByCompetition_Id(Integer id);

```

Comment:
Skoro to lista to nie musi być optionalem - w najgorszym przypadku będzie to pusta lista a nie null z tego co pamiętam. Dodatkowo findByCompetitionId powinno zadziałać - zapis COMPETITION_ID z bazy dany powinien być z automatu przerobiony na competitionId więc nie powinno być problemów - jeśli są to powinieneś się zainteresować ustawieniami hibernate w springu tj. flagi NamingStrategy